exp/addons/api/v1alpha3/clusterresourceset_types.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ClusterResourceSetSecretType is the only accepted type of secret in resources
+	ClusterResourceSetSecretType corev1.SecretType = "clusterResourceSet"
```

Comment:
```suggestion
	ClusterResourceSetSecretType corev1.SecretType = "addons.cluster.x-k8s.io/resource-set"
```

As @ncdc suggested, just wanted to capture the conversation and pros/cons of this approach

Reply:
Changed this to the format you suggested for now. Let's see if there are any concerns.